torch/quantization/_numeric_suite_fx.py

```diff
@@ -1,27 +1,22 @@
 from typing import Any, Dict
 
 import torch
-import torch.nn as nn
-import torch.nn.quantized as nnq
-import torch.nn.quantized.dynamic as nnqd
 from torch.fx import GraphModule  # type: ignore
 from torch.fx import map_arg  # type: ignore
 from torch.fx.graph import Graph
+from torch.quantization import get_default_compare_output_module_list
 from torch.quantization._numeric_suite import (
+    _find_match,
     get_logger_dict,
     prepare_model_with_stubs,
     compare_weights,
+    Logger,
+    OutputLogger,
     ShadowLogger,
 )
+from torch.quantization.fx.quantization_patterns import NumericSuiteQuantizeHandler
```

Comment:
can you define this handler in a different file?

Reply:
Make sense. moving to numeric suite